dataactvalidator/config/sqlrules/b9_award_financial.sql

```diff
@@ -28,7 +24,8 @@ WHERE af.submission_id = {0}
                 AND af.main_account_code IS NOT DISTINCT FROM pa.account_number
                 AND LOWER(af.program_activity_name) IS NOT DISTINCT FROM pa.program_activity_name
                 AND af.program_activity_code IS NOT DISTINCT FROM pa.program_activity_code
-                AND CAST(pa.budget_year as integer) = (SELECT reporting_fiscal_year
-                                                            FROM submission
-                                                            WHERE submission_id = af.submission_id))
-	);
\ No newline at end of file
+                AND (CAST(pa.budget_year as integer) = 2016
```

Comment:
You could also do something like:
```
CAST(pa.budget_year as integer) in (2016, (SELECT reporting_fiscal_year
						  FROM submission
						  WHERE submission_id = af.submission_id))
```

Not sure if reducing to use a single CAST for `budget_year` would affect performance, but this is just another option to checking multiple years.